recipe/meta.yaml

```diff
@@ -9,8 +9,10 @@ source:
   sha256: d6fac2342802c30e51426828fe084ff4deb1b3387367cf98976bb2e64b6f8e45
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 2
+  script:
+    - ls "${PREFIX}/bin"  # [unix]
```

Comment:
Oops sorry didn't expect this to get merged so quickly. Removing in PR ( https://github.com/conda-forge/cython-feedstock/pull/91 )